lab_3_keywords_textrank/main.py

```diff
@@ -51,7 +52,11 @@ def _clean_and_tokenize(self, text: str) -> tuple[str, ...]:
             tuple[str, ...]
                 clean lowercase tokens
         """
-        pass
+        clean_text = []
+        for symbol in text:
+            if symbol not in self._punctuation:
+                clean_text += symbol
+        return tuple(" ".join(clean_text).lower().split())
```

Comment:
better to save `" ".join(clean_text).lower().split()` to a variable and then cast to tuple - difficult to read